content/blog/2025-04-07-spring-2025-raspberrypi-updates.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Spring 2025 Raspberry Pi Updates"
+type: blog
+author:
+  name: "Koichiro Iwao"
+  bio: "Engineer at Cybertrust Japan"
+  image: /users/koichiroiwao.jpg
+date: '2025-04-07'
+#images:
+post:
+  title: "Spring 2025 Raspberry Pi Updates"
+---
+
+Hello, Community! I'm pleased to share the latest updates on Raspberry Pi with you. Have fun on the Pi!
+
+## AlmaLinux OS Kitten 10 is now available for Raspberry Pi
+
+As you might have already noticed, AlmaLinux OS Kitten 10 is now available for Raspberry Pi! Visit [AlmaLinux OS Kitten 10 page](https://wiki.almalinux.org/development/almalinux-os-kitten-10.html#raspberry-pi) for details.
+
+Both the console image and the GNOME image are available, along with previous versions; however, the MBR version has been omitted to reduce development effort. Therefore, only images with a GUID partition table are available in AlmaLinux OS Kitten 10, which means that the Raspberry Pi 3+ is no longer supported.
```

Comment:
```suggestion
Both the console image and the GNOME image are available, as with previous versions; however, the MBR version has been omitted to reduce development effort. Therefore, only images with a GUID partition table are available in AlmaLinux OS Kitten 10. That means that the Raspberry Pi 3+ will not be supported for AlmaLinux OS 10 or AlmaLinux OS Kitten 10
```
We should clarify that this is only for 10. If it's true for older versions as well, we'll need to adjust messaging further.

Reply:
Indeed.